larpandora/LArPandoraInterface/LArPandoraGeometry.cxx

```diff
@@ -361,12 +337,11 @@ void LArPandoraGeometry::LoadGeometry(LArDriftVolumeList &driftVolumeList)
             }
 
             // Create new daughter drift volume (volume ID = 0 to N-1)
-            if (!isDualPhase)
-                driftVolumeList.emplace_back(LArDriftVolume(driftVolumeList.size(), isPositiveDrift,
-                    wirePitchU, wirePitchV, wirePitchW, wireAngleU, wireAngleV, wireAngleW,
-                    0.5f * (driftMaxX + driftMinX), 0.5f * (driftMaxY + driftMinY), 0.5f * (driftMaxZ + driftMinZ),
-                    (driftMaxX - driftMinX), (driftMaxY - driftMinY), (driftMaxZ - driftMinZ),
-                    (wirePitchU + wirePitchV + wirePitchW + 0.1f), tpcVolumeList));
+            driftVolumeList.emplace_back(LArDriftVolume(driftVolumeList.size(), isPositiveDrift,
+                wirePitchU, wirePitchV, wirePitchW, wireAngleU, wireAngleV, wireAngleW,
+                0.5f * (driftMaxX + driftMinX), 0.5f * (driftMaxY + driftMinY), 0.5f * (driftMaxZ + driftMinZ),
+                (driftMaxX - driftMinX), (driftMaxY - driftMinY), (driftMaxZ - driftMinZ),
+                (wirePitchU + wirePitchV + wirePitchW + 0.1f), tpcVolumeList));
```

Comment:
My main comment from last time was that by using `emplace_back` you do not need to additionally specify the type you are inserting--the compiler already knows what it is.  For example:
```.cpp
driftVolumeList.emplace_back(driftVolumeList.size(), isPositiveDrift, ..., tpcVolumeList);
```
is equivalent to:
```.cpp
driftVolumeList.push_back(LArDriftVolume(driftVolumeList.size(), isPositiveDrift, ..., tpcVolumeList));
```
No reason to hold up approval of the PR, but just an FYI.